pyspark-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/hooks.py

```diff
@@ -16,7 +16,7 @@ def after_context_created(self, context) -> None:
 
         # Initialise the spark session
         spark_session_conf = (
-            SparkSession.builder.appName(context._package_name)
+            SparkSession.builder.appName(context._project_path.name)
```

Comment:
Seems like there's a public, read-only `project_path` property? https://docs.kedro.org/en/stable/kedro.framework.context.KedroContext.html#kedro.framework.context.KedroContext.project_path

Reply:
That's a good point. Let's use that one instead.

Reply:
Good catch; thanks!